escape_roomba/thread_manager.py

```diff
@@ -80,101 +59,80 @@ async def _on_ready(self):
-            await self._async_check_message(
+        if p.user_id != self._context.discord().user.id:
+            await self._async_message_update(
                 channel_id=p.channel_id, message_id=p.message_id, emoji=p.emoji)
 
     async def _on_raw_reaction_remove(self, p):
-        if p.user_id != self._context.client.user.id:
-            await self._async_check_message(
+        if p.user_id != self._context.discord().user.id:
+            await self._async_message_update(
                 channel_id=p.channel_id, message_id=p.message_id, emoji=p.emoji)
 
     async def _on_raw_reaction_clear(self, p):
-        await self._async_check_message(
+        await self._async_message_update(
             channel_id=p.channel_id, message_id=p.message_id)
 
     async def _on_raw_reaction_clear_emoji(self, payload):
-        await self._async_check_message(
+        await self._async_message_update(
             channel_id=p.channel_id, message_id=p.message_id, emoji=p.emoji)
 
     async def _on_guild_channel_delete(self, channel):
         await self._async_forget_channel(channel)
 
     async def _on_guild_channel_create(self, channel):
-        await asyncio.gather(self._async_check_channel(channel),
+        await asyncio.gather(self._async_channel_update(channel),
                              self._async_fetch_recents(channel))
 
     async def _on_guild_channel_update(self, before, after):
-        await self._async_check_channel(after)
+        await self._async_channel_update(after)
 
     #
     # Internal methods
     #
     # Operations lock self._exclusive_message using the origin message ID (int)
     # of the thread (or thread-to-be) to avoid races.
     #
-    # Discord message event listeners (above)
-    #  🡲  _async_check_message (filters update, acquires lock)
-    #     ┄┄┄┄┄┄┄ methods below run with the lock held ┄┄┄┄┄┄┄┄
-    #      🡢  ( _async_fetch_locked_intro for intro changes - see below )
-    #      🡲  _async_fetch_locked_message (attempts message fetch)
-    #          🡲  ⎛ _async_locked_message_update (if fetch successful)    ⎞
-    #             ⎝ _async_locked_message_gone (if fetch was "not found") ⎠
-    #                   ⎛ _async_create_thread_for_locked_message (if new 🧵) ⎞
-    #                🡲  ⎜ _async_delete_locked_thread (if 🧵/message gone)    ⎟
-    #                   ⎝ _async_update_locked_intro (if new 🧵/message edit) ⎠
-    #
-    # Discord channel event listeners (above)
-    #  🡲  _async_fetch_recents (catches up to recent channel history on start)
-    #      🡢  ( _async_check_message for recent messages - see above )
-    #  🡲  _async_forget_channel (removes _Thread object from tracking)
-    #  🡲  _async_check_channel (filters update, acquires lock)
-    #     ┄┄┄┄┄┄┄ methods below run with the lock held ┄┄┄┄┄┄┄┄
-    #      🡲  _async_fetch_locked_intro (loads first 2 messages)
-    #          🡢  ( _async_fetch_locked_message - see section above )
-    #
```

Comment:
Did this documentation disappear altogether? Maybe it's my 10:30 PM brain, but I don't see it, not sure if that was intentional.